README.md

```diff
@@ -14,17 +14,42 @@ go get github.com/jrperritt/rack
 go build -o $GOPATH/bin/rack
 ```
 
+## Setting Authentication Credentials
+
+### Environment Variables
 Export the following environment variables:  
-RS_REGION_NAME (DFW, IAD, ORD, LON, SYD, HKG)  
-RS_USERNAME (Your Rackspace username)  
-RS_AUTH_URL (https://identity.api.rackspacecloud.com/v2.0)  
-RS_API_KEY (Your Rackspace API key)  
+`RS_REGION_NAME` (DFW, IAD, ORD, LON, SYD, HKG)  
+`RS_USERNAME` (Your Rackspace username)  
+`RS_API_KEY` (Your Rackspace API key)  
```

Comment:
Thanks for formatting these. They were driving me a bit bonkers every time I skimmed the README.